cfbs/main.py

```diff
@@ -118,6 +123,7 @@ def main() -> int:
 
     # Commands you can run outside a cfbs repo:
     if args.command == "help":
```

Comment:
I like how we still haven't agreed on a single-working-everywhere command to show help.

Is it `cmd -?`
or `cmd -h`?
or `cmd --help`?
or more pytonic `cmd help`.

\</rant>